src/components/molecules/RadioGroupField/RadioGroupField.md

```diff
@@ -0,0 +1,78 @@
+### Summary
+
+`<RadioFieldGroup />` is a convenient wrapper that renders group of `<RadioField>`s:
+
+⚠️ &nbsp;Note that the `value` for each item has to be unique as it determines the `id`
+and `htmlFor` prop fof each radio field.
+
+### Examples
+
+- Uncontrolled
+
+```jsx
+import { RadioGroupField } from '@zopauk/react-components';
+
+<RadioGroupField
```

Comment:
Should we take the opportunity to add `<Legend />` and `<FieldSet />` to these examples as well?  👨🏻‍🏫

In this way people is gonna be more mindful to use them, another option is to state that is important to use them in `## Summary` ( _linking to the respective docs sections_ )

Reply:
`<RadioGroupField />` already uses `<Legend />` and `<Fieldset />` internally